howto/anbox/harden.md

```diff
@@ -0,0 +1,72 @@
+(howto-harden)=
+# How to harden your deployment
+
+Anbox Cloud is designed with secure development practices but there are certain aspects to consider on your side to make sure the entire deployment is secure.
+
+## General guidelines
+
+Consider the following simple yet impactful measures to ensure that you run a secure Anbox Cloud deployment:
+
+- Always run the latest and supported version of Anbox Cloud. See {ref}`ref-release-notes`.
```

Comment:
I feel like it'd be great if we could mention the `upgrade scenario` where people need to keep their deployment up to date?
If that stands,
```
Always run the latest and supported version of Anbox Cloud. See {ref}`ref-release-notes`.
```
->
```
Always run the latest supported version of Anbox Cloud and keep your deployment up to date.
 See {ref}`ref-release-notes`.
```